README.rst

```diff
@@ -33,8 +33,8 @@ A Concrete Syntax Tree (CST) parser and serializer library for Python
 
 .. intro-start
 
-LibCST parses Python 3.5, 3.6, 3.7 or 3.8 source code as a CST tree that keeps all
-formatting details (comments, whitespaces, parentheses, etc). It's useful for
+LibCST parses Python 3.0, 3.1, 3.3, 3.5, 3.6, 3.7 or 3.8 source code as a CST tree that keeps
```

Comment:
nit: I would reword this to say
```suggestion
LibCST parses source code compatible with all Python versions between 3.0 and 3.8 (inclusive) as a CST tree that keeps
```

Reply:
Python 3.2 and 3.4 are not mentioned here, assuming they don't have new syntax change?

Reply:
@jimmylai there were no grammar changes in 3.2 or 3.4, so I didn't include them (they would be duplicates of the 2.1 and 3.3 grammar if allowed). My intent with KNOWN_PYTHON_VERSION_STRINGS is that you could do

```
for v in reversed(KNOWN_PYTHON_VERSION_STRINGS):
  try:
    mod = parse(...python_version=v)
    return mod
  except ParseError:
    pass
```

to see if it parses on any.  I'm happy to add some helpers here (e.g. if you have a `/usr/bin/pythonx.y` and want to pick an ideal grammar version.